src/index.html

```diff
@@ -18,10 +18,10 @@
           <h1 class="logo__title"><span>Mamma</span>Mia!</h1>
           <h2 class="logo__subtitle">the best homemade Italian food</h2>
         </div>
-        <div id="cart" class="cart">
+        <div id="toggle-cart" class="cart">
```

Comment:
1. Tutaj powinien zostać identyfikator `cart`
```suggestion
        <div id="cart" class="cart">
```